common-pipeline/operators/conditions-v3-schema.json

```diff
@@ -350,6 +358,12 @@
                 },
                 {
                   "type": "boolean"
+                },
+                {
+                  "type": "array"
+                },
+                {
+                  "type": "null"
```

Comment:
This was added to support checking null values for numbers, and empty arrays for lists/structures